template/src/ontology/run.sh.jinja2

```diff
@@ -8,9 +8,16 @@
 # The assumption is that you are working in the src/ontology folder;
 # we therefore map the whole repo (../..) to a docker volume.
 #
+# To use singularity instead of docker, please issue
+#
+# export USE_SINGULARITY=<any-value>
+#
+# before running this script.
+#
```

Comment:
Collapse the above to one row of comment.

Reply:
Addressed in [d5d5513](https://github.com/INCATools/ontology-development-kit/pull/640/commits/d5d5513920b74e9729e7c3f173b9064f480d1621).